### DISCUSSION TITLE
Baggage support is confusing

### DISCUSSION THREAD

**Discussion by dominikjeske** [dotnet/aspnetcore#54821] [General]
From my tests on [Baggage header](https://opentelemetry.io/docs/concepts/signals/baggage/) it looks very confusing especially when OTel is used. I put my report in this issue https://github.com/dotnet/runtime/issues/45496#issuecomment-2024691280 because problem is in ASP+HttpClient+OTel. For me the best would be either abend 'Correlation-Context' or make it configurable to use either 'Correlation-Context' or  'baggage' but not both.

**Comment by gfoidl** [comment]
@JamesNK please have a look here.
